### PR TITLE
[Varbase SEO] Optimization round to have better init Metatag configs

### PR DESCRIPTION
**Pull request by yasmeensalah**

### CODE REVIEW

File: modules/varbase_features/varbase_core/varbase_core.tokens.inc [hunk 1]
```diff
@@ -0,0 +1,47 @@
+<?php
+
+use Drupal\Core\Render\BubbleableMetadata;
+
+/**
+ * Implements hook_token_info().
+ */
+function varbase_core_token_info() {
+  // Current theme token.
+  $info['types']['current-theme'] = array(
```
Comment (moerazem) [inline modules/varbase_features/varbase_core/varbase_core.tokens.inc]: default-active-theme

File: modules/varbase_features/varbase_core/varbase_core.tokens.inc [hunk 2]
```diff
@@ -0,0 +1,47 @@
...
+    'name' => t('Current theme'),
```
Comment (moerazem) [inline modules/varbase_features/varbase_core/varbase_core.tokens.inc]: Default theme

File: modules/varbase_features/varbase_seo/varbase_seo.install [hunk 3]
```diff
@@ -103,3 +105,23 @@ function varbase_seo_update_8011() {
...
+  $profile_path = drupal_get_path('module', 'varbase_seo') . '/config/install/';
+  $configs['metatag.metatag_defaults.403'] = $profile_path . 'metatag.metatag_defaults.403.yml';
+  $configs['metatag.metatag_defaults.404'] = $profile_path . 'metatag.metatag_defaults.404.yml';
+  $configs['metatag.metatag_defaults.front'] = $profile_path . 'metatag.metatag_defaults.front.yml';
+  $configs['metatag.metatag_defaults.global'] = $profile_path . 'metatag.metatag_defaults.global.yml';
+  $configs['metatag.metatag_defaults.node'] = $profile_path . 'metatag.metatag_defaults.node.yml';
+  $configs['metatag.metatag_defaults.taxonomy_term'] = $profile_path . 'metatag.metatag_defaults.taxonomy_term.yml';
+  $configs['metatag.metatag_defaults.user'] = $profile_path . 'metatag.metatag_defaults.user.yml';
+
+  foreach ($configs as $name => $config_path) {
```
Comment (moerazem) [inline modules/varbase_features/varbase_seo/varbase_seo.install]: Make sure to not Override existing conf.

**Review by moerazem — DISMISSED** [review]

**Review by Natshah — DISMISSED** [review]
Approved

**Pull request merged**